### PR TITLE
ISSUE_TEMPLATE: firm up language

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Try to be even clearer that we need you to read this.

### CODE REVIEW

**Review by WesWedding — CHANGES_REQUESTED** [review]
"If you do not read fill out the issue template repeatedly you will be blocked from every submitting issues to Homebrew again."

This particular sentence has an issue:
 "If you do not read fill out" seems like a typo.  Maybe it was supposed be "If you do not fill out"? Or "if you do not read/fill out"

Alternate wording might be "Repeated failure to use the issue template will result in a block from the Homebrew issue queue."  Sounds almost like legalese and extra scary!

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
+**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you do not read fill out the issue template repeatedly you will be blocked from every submitting issues to Homebrew again.**
+
```
Comment (chdiza) [inline .github/ISSUE_TEMPLATE.md]: "ever submitting"

**Comment by DomT4** [comment]
Please note this is not disagreement with this PR, because lord I have not forgotten how frustrating handling issues with almost zero basic information was, but I think it will always be the case that people retain that behaviour even if Homebrew suggests it might chuck them into the ocean without a lifejacket for doing so. I hope this decreases the prevalence but I fear otherwise 😭.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 To help us debug your issue please explain:
 - What you were trying to do (and why)
 - What happened (include command output)
 - What you expected to happen
 - Step-by-step reproduction instructions (by running `brew install` commands)
 
-# New Formulae
-To get a new formula into Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md).
+# New or Updated Formulae
+To get a new formula into Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Issues created requesting a new/updated formulae will be closed.
```
Comment (vszakats) [inline .github/ISSUE_TEMPLATE.md]: "To get a new formula into" -> ~"To get formula changes into"

**Comment by MikeMcQuaid** [comment]
> Please note this is not disagreement with this PR, because lord I have not forgotten how frustrating handling issues with almost zero basic information was, but I think it will always be the case that people retain that behaviour even if Homebrew suggests it might chuck them into the ocean without a lifejacket for doing so.

Yup, I agree. Going beyond this, though, requires GitHub to do stuff (and this is something I'm advocating for internally).

**Comment by MikeMcQuaid** [comment]
Thanks for the review everyone 🎉

**Pull request merged**